### ISSUE TITLE
how can i show the real time accuracy when i am trianing the network??

### ISSUE THREAD

**Issue by minmummax** [davisking/dlib#821]
I can show the real time accuracy when I am training my network the tensorflow framework.
but how can I show it with dlib??
should I change the trainer.be_verbose() function??or everytime I save the net's parameter to test it??
give me a method to solve this problem thanks !!

**Comment by davisking** [comment]
The verbose output includes the current loss which is probably what you mean by "real time accuracy".  If you want to print some other information then write some code that prints it periodically.

**Comment by minmummax** [comment]
the loss does not mean the accuracy I think,
loss depends on the training sets;
but the accuracy depends on my test sets.

**Comment by davisking** [comment]
Then call test_one_step() or periodically do some tests on your test set and print your evaluation metric.

**Comment by minmummax** [comment]
so at last I must write the test function to realize this.
